fe/fe-core/src/main/java/com/starrocks/alter/Alter.java

```diff
@@ -297,6 +311,14 @@ public void processAlterMaterializedView(AlterMaterializedViewStmt stmt)
                 } catch (AnalysisException ae) {
                     throw new DdlException(ae.getMessage());
                 }
+            } else if (status != null) {
+                if ("active".equalsIgnoreCase(status) && materializedView.isActive()) {
+                    throw new DdlException("Materialized view [" + materializedView.getName() + "] is already active");
```

Comment:
how about making it idempotent and return success

Reply:
Yes, I don't know if idempotent is good or give a message, maybe we can discuss it